modules/tensor_mechanics/tests/Wave_1_D/HHT_time_integration/tests

```diff
@@ -0,0 +1,8 @@
+[tests]
+  [./test]
+    type = 'Exodiff'
+    input = 'wave_bc_1d.i'
```

Comment:
1D is written without a space (or an underscore). I'd suggest to put all those tests inside a ```dynamic_mechanics``` folder and flatten the hierarchy within that a bit (per my previous comment).